src/CommandingOfficers/AWBW/GE/Drake.java

```diff
@@ -0,0 +1,164 @@
+package CommandingOfficers.AWBW.GE;
+
+import java.util.ArrayList;
+import CommandingOfficers.*;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.GlobalWeatherEvent;
+import Engine.GameEvents.MassDamageEvent;
+import UI.UIUtils;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Terrain.Environment.Weathers;
+import Units.Unit;
+import Units.UnitContext;
+
+public class Drake extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Drake", UIUtils.SourceGames.AWBW, UIUtils.GE);
+      infoPages.add(new InfoPage(
+            "Drake (AWBW)\n"
+          + "aval units gain +1 movement and +10 defense. Air units lose -30% attack. Unaffected by rain (except vision), and has a higher chance of Rain in random weather.\n"));
```

Comment:
'N' <-- Here, you dropped this.

Reply:
Also your capitalization of "rain" is inconsistent.

Reply:
Ain't my capitalization https://awbw.amarriner.com/co.php

Reply:
Well, FINE then.